languages/go/exercises/concept/using-map/usingmap.go

```diff
@@ -0,0 +1,18 @@
+package usingmap
+
+// var priceTable
+
+// TransactionLog represent a single item in a transaction
+type TransactionLog struct {
+	itemName string
+	month    string
+	qty      int
+}
+
+func calculateTotalIncome(monthlyIncome map[string]int) int {
```

Comment:
I'd suggest to have this function exported: `CalculateTotalIncome` and put a comment on it explaining what is does. That is an additional help to the student implementing the function and also shows how functions in Go are properly commented.

```suggestion
// CalculateTotalIncome sums up the total income from the priceTable.
func CalculateTotalIncome(monthlyIncome map[string]int) int {
```